### ISSUE TITLE
About de-blurring training

### ISSUE THREAD

**Issue by zhunzhun123** [swz30/MPRNet#129]
Thanks for your code, I'll outline the problem I encountered

1) In the training process, I adjusted batch-size to 2, and the training data set was 600 pairs of photos. Why is there a great loss after the training? The loss after the training is more than 20.

2) Another question is why I can't find my training log after training.

Hope to get the author's answer.
